test/font_test.py

```diff
@@ -118,7 +118,10 @@ def test_match_font_all_exist(self):
         for font in fonts:
             path = pygame_font.match_font(font)
             self.assertFalse(path is None)
-            self.assertTrue(os.path.isabs(path))
+            self.assertTrue(
+                os.path.isabs(path) and
+                os.path.isfile(path)
+            )
```

Comment:
Super minor thing, black is going to complain about the formatting here (lint CI will fail if I re-run it).
This needs to be on one line because it is within the line limit